src/steamship_langchain/vectorstores/steamship_vector_store.py

```diff
@@ -73,7 +98,45 @@ def __init__(
             fetch_if_exists=True,
         )
 
-    def add_texts(self, texts: Iterable[str], metadatas: Optional[List[dict]] = None) -> None:
+    def add_files(
+        self, files: Iterable[File], splitter: Optional[TextSplitter] = None
+    ) -> List[str]:
+        """Run Steamship Files through the embeddings and add to the vectorstore.
+
+        Args:
+            files: collection of Steamship Files to add to the VectorStore
+            splitter: optional splitter to use to trim text content of Files into reasonable chunks
+
+        Returns:
+            List of ids from adding the files into the VectorStore.
+        """
+        ids = []
+        for file in files:
+            source_texts = []
+            for block in file.blocks:
+                if splitter:
+                    source_texts.extend(splitter.split_text(block.text))
+                else:
+                    source_texts.append(block.text)
+
+            texts = [_sanitize_text(t) for t in source_texts if len(t) > 0]
+            if len(texts) == 0:
+                continue
```

Comment:
This is considered an anti-pattern in Python. 

Would either ignore this case and let the code deal with empty strings OR flip the if statement to len(texts) != 0:

Reply:
This = Use of `continue` in Python

Reply:
Refactored. I find it strange that python prefers indentation to `continue` in loops, but when in ancient rome :).